.env.example

```diff
@@ -106,3 +106,8 @@ GITHUB_TOKEN=
 ## Kubernets sidecar
 # SECRET_SIDECAR_IMAGE= # optional, path to the docker image that built from the secret_puller service.  service will be disabled w/o it
 
+
+## Jenkins, required for triggering Jenkins build after deployment
```

Comment:
we try to use the `required` work for things that need to be filled out, how about `Jenkins, optional for triggering Jenkins build after deployment`